### PR TITLE
improve BLAST page + add nix flake

### PR DESCRIPTION
**Pull request by williambl**
The Nix flake allows NixOS users to run the site locally with just `nix develop` and `bundle exec jekyll serve`.

The updated BLAST page looks like this:
image

### CODE REVIEW

**Review by Pyrofab — CHANGES_REQUESTED** [review]
The Nix stuff seems useful if not a bit niche, but unless I am mistaken you need to add the files to Jekyll's exclusion list in the config

**Comment by williambl** [comment]
Ah, thanks - done that now :)

File: wiki/blast.md [hunk 1]
```diff
@@ -24,7 +24,57 @@ BLAST mainly focuses on various types of bombs with some common attributes:
...
+
+<p>A version of the golden bomb with a trigger instead of a fuse.</p>
+
+<p>It also expldoes underwater without causing block destruction.</p>
+</td><td>
+<h4>Diamond Trigger Bomb</h4>
+
+<p>A version of the diamond bomb with a trigger instead of a fuse.</p>
+
+<p>It also expldoes underwater without causing block destruction.</p>
```
Comment (Pyrofab) [inline wiki/blast.md]:
```suggestion
<p>It also explodes underwater without causing block destruction.</p>
```

**Review by sschr15 — APPROVED** [review]
cool beans

File: wiki/blast.md [hunk 2]
```diff
@@ -24,7 +24,57 @@ BLAST mainly focuses on various types of bombs with some common attributes:
...
+
+<p>A simple bomb with an explosion power of 1 on impact instead of after a certain amount of time.</p>
+
+<p>Explodes underwater, but will not destroy any blocks.</p>
+</td><td>
+<h4>Golden Trigger Bomb</h4>
+
+<p>A version of the golden bomb with a trigger instead of a fuse.</p>
+
+<p>It also expldoes underwater without causing block destruction.</p>
```
Comment (Pyrofab) [inline wiki/blast.md]:
```suggestion
<p>It also explodes underwater without causing block destruction.</p>
```

**Review by Pyrofab — CHANGES_REQUESTED** [review]
expldoes

**Review by Pyrofab — APPROVED** [review]
Seems good to me, thank you !

**Comment by Xiribidus** [comment]
looks funny, i like




nya

Credit for this wonderful review goes to Pyrofab, who inspired me to do this

**Comment by UpcraftLP** [comment]
LGTM 👍🏻

**Review by Xiribidus — APPROVED** [review]
ok ok fr now

File: wiki/blast.md [hunk 3]
```diff
@@ -121,7 +171,7 @@ Both the normal and fresh variant can be converted to Dried Folly Red Paint, whi
 ## Claim / Protection Mod Support
 
 Blast version 1.12 and above has support for Patbox's [Common Protection API](https://github.com/Patbox/common-protection-api).
-This means BLAST itema and blocks are un-able to bypass claim protections for any mod that implements this common API.
```
Comment (EvieTheOwl) [inline wiki/blast.md:124]: Apologies for my atrocious spelling & grammar lol and thank u for noticing ☻

**Pull request merged**